cloudencrypt/metadata.go

```diff
@@ -0,0 +1,14 @@
+package cloudencrypt
+
+type MetadataKV struct {
+	Key   string
+	Value string
+}
+
+func buildMetadataMap(metadata ...MetadataKV) map[string]string {
```

Comment:
This seems to be internal only

Reply:
well yes, that's why it's not exported...

Reply:
Throw it under internal where library specific things are kept. Even though they are public functions/struct